### DISCUSSION TITLE
Integrated a script inline of phtml file

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#204] [3. Integrations]
Hi dears i want to integrate a module for breeze theme. my code like below but when i enable breeze doesn't added to page
Please Help!

```
<script type="text/x-magento-init">
            {
                "*": {
                    "Magento_Customer/js/block-submit-on-send": {
                        "formId": "login-form"
                    },
                    "Magento_Ui/js/core/app": {
                        "components": {
                            "showPassword": {
                                "component": "Magento_Customer/js/show-password",
                                "passwordSelector": "#pass"
                            }
                        }
                    }
                }
            }
        </script>
```

**Comment by vovayatsyuk** [comment]
You need to register your components. Please read the [integration example](https://breezefront.com/docs/integration-example)

**Reply by ppeyman** [reply]
Hi dear @vovayatsyuk  i registerd it and file loaded 
but i don't know why my file doesn't run well.
I would be grateful if you could Check [My File](https://products.groupdocs.app/editor/source/custom-login.js/29527084-f015-4c8b-860a-c10d8bee33bf) .

**Reply by vovayatsyuk** [reply]
Do you use developer console in your browser? Does it show you some errors? Can you add the link to your site?

**Reply by ppeyman** [reply]
yes i check console doesn`t any errors 
site in my local i work on it
would you i send module for you ?

**Reply by vovayatsyuk** [reply]
No, I can check what's wrong if you'll give me the link to your site.

p.s. The component name you use in js code is `Magedelight_SMSProfile/js/custom-login`. In your first message, I didn't see this component in HTML code.

**Reply by ppeyman** [reply]
dear @vovayatsyuk 
i install it on online server please check :

[Here](http://starbox.run.place/customer/account/login/referer)

thank you

**Reply by vovayatsyuk** [reply]
Your widgets are using the same name:

```js
$.widget('mage.mdLoginOTP', {
        component: 'Magedelight_SMSProfile/js/custom-login',
});

$.widget('mage.mdLoginOTP', {
        component: 'Magedelight_SMSProfile/js/otpcode',
});
```

The name must be unique.
